tests/user.test.ts

```diff
@@ -1,47 +1,52 @@
-import { GraphQLClient, request } from 'graphql-request';
+import { getClientUserSignedIn, testHost } from './testSetup';
 
-import { testHost } from './testSetup';
+import { ErrorString } from '../src/utils/error';
+import models from '../src/models';
+import { request } from 'graphql-request';
 
 jest.mock('../src/utils/virgil', () => {
   return jest.fn();
 });
 
-describe('Resolver - User', () => {
-  const name = 'dooboo1';
-  const email = `${name}@dooboo.com`;
-  const signUpUser = /* GraphQL */`
-  mutation signUp($user: UserInput!) {
-    signUp(user: $user) {
-      token,
+const signInEmail = /* GraphQL */ `
+  mutation {
+    signInEmail(email: "dooboo1@dooboo.com", password: "password") {
+      token
       user {
-        id
         email
       }
     }
   }
 `;
 
-  it('should signUp user', async () => {
-    const response = await request(testHost, signUpUser, {
-      user: {
-        email,
-        password: 'password',
-        name: 'test-1',
+const verifyUsers = async (): Promise<void> => {
+  /* Verify Users */
+  const { User: userModel } = models;
+  await userModel.update(
+    {
+      verified: true,
+    },
+    {
+      where: {
+        verified: false,
       },
-    });
+    },
+  );
+};
 
-    expect(response).toHaveProperty('signUp');
-    expect(response.signUp).toHaveProperty('token');
-    expect(response.signUp).toHaveProperty('user');
-    expect(response.signUp.user.email).toEqual(email);
-  });
+describe('Resolver - User', () => {
+  const name = 'dooboo1';
+  const email = `${name}@dooboo.com`;
+  const password = 'password';
+  const updateName = 'geoseong';
 
-  const signInEmail = /* GraphQL */`
+  const signUp = /* GraphQL */ `
```

Comment:
Looks like we can make enhancement with this mutation query followed by [here](https://github.com/dooboolab/hackatalk-server/blob/34d90c37b57bad3defe37ff95fc62e76706288e9/tests/message.test.ts#L9-L28).

Reply:
ouch, I missed that 😅